### ISSUE TITLE
ArchitectureCraft incompatibility

### ISSUE THREAD

**Issue by zcrafter1337** [sp614x/optifine#1511]
ArchitectureCraft blocks are rendered invisible or with unicolor-textures when using optifine in MC 1.12.2.

**Comment by sp614x** [comment]
ArchitectureCraft is only updated to 1.10.2.
https://minecraft.curseforge.com/projects/architecturecraft/files

Do you mean the ElytraDev Alpha?
https://minecraft.curseforge.com/projects/architecturecraft-elytradev/files

**Comment by sp614x** [comment]
Testing Forge #2729 + OptiFine E2_pre + ArchitectureCraft ElytraDev Alpha 1.12-2.68.

Seems to work correctly:

![image](https://user-images.githubusercontent.com/16635441/42379809-433903a2-812c-11e8-8069-9efc528b07c5.png)

Also with shaders:

![image](https://user-images.githubusercontent.com/16635441/42380020-ce2ad742-812c-11e8-9eff-aa2a291f1b04.png)

**Comment by sp614x** [comment]
Breaks when Antialiasing or Anisotropic Filtering is enabled:

![image](https://user-images.githubusercontent.com/16635441/42380084-04a98232-812d-11e8-9956-8ca2811f8164.png)

**Comment by zcrafter1337** [comment]
> Do you mean the ElytraDev Alpha?

Yes you are correct. Sorry for not posting the exact version number. I noticed that issue while playing the FTB Revelation Pack v2.1.0.
